py/embedded-server/README_PyPi.md

```diff
@@ -0,0 +1,27 @@
+# deephaven-server
+
+The Dockerless version of [deephaven-core](https://www.github.com/deephaven/deephaven-core).
+
+Deephaven Community Core is a real-time, time-series, column-oriented analytics engine with relational database features. It is an open version of [Deephaven Enterprise](https://deephaven.io/enterprise/), which serves as the data backbone for prominent hedge funds, banks, and financial exchanges.
+
+`deephaven-server` enables users to run Deephaven Community Core without Docker.
+
+## Setup
+
+Java 11+ is required for this module, and the `JAVA_HOME` environment variable must be set appropriately.  Once that's done, you can use pip:
```

Comment:
I don't like attaching this as a *setup* requirement - it's not necessary for pip installs. That said, it is important info, and having it set during pip install won't hurt.

I'd suggest creating a "## Requirements" header before Setup; where we can also mention that it requires Python 3.7+.